persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -60,12 +61,16 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor wit
 
     case ReadHmdaRawFile(persistenceId) =>
       publishEvent(ParsingStarted(submissionId))
+      var encounteredParsingErrors: Boolean = false
```

Comment:
Actors are one of the few places where using `var` is safe, since they are designed to keep state. I think it would probably be safest to put this variable at the top level, so it gets initialized (i.e. to false) and it's then changed accordingly as messages flow through the `receive` method